.goreleaser.yml

```diff
@@ -35,3 +35,5 @@ brews:
     homepage: https://github.com/cirruslabs/tart
     description: Run macOS VMs on Apple Silicon
     skip_upload: auto
+    custom_block: |
+      depends_on :macos => :monterey
```

Comment:
Is it just Monterey or Monterey and newer?

Reply:
It seems to be the latter, can be validated with `depends_on :xcode => "15.0"` and the docs also say:

>Some [Requirement](https://rubydoc.brew.sh/Requirement)s can also take a string specifying their minimum version that the formula depends on.

(https://docs.brew.sh/Formula-Cookbook#specifying-other-formulae-as-dependencies)